element_calcium_imaging/scan.py

```diff
@@ -606,18 +606,20 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
                 )
 
                 self.Field.insert(
-                    dict(
-                        key,
-                        field_idx=plane_index,
-                        px_height=pvscan_info["height_in_pixels"],
-                        px_width=pvscan_info["width_in_pixels"],
-                        um_height=pvscan_info["height_in_um"],
-                        um_width=pvscan_info["width_in_um"],
-                        field_x=pvscan_info["fieldX"],
-                        field_y=pvscan_info["fieldY"],
-                        field_z=pvscan_info["fieldZ"][plane_index],
-                    )
-                    for plane_index in range(pvscan_info["num_planes"])
+                    [
```

Comment:
You don't need to turn this into a list, datajoint `.insert()` can handle a generator fine